nat-lab/docker-compose.yml

```diff
@@ -414,6 +418,8 @@ services:
     volumes:
       - ./data/nordderper/config1.yml:/etc/nordderper/config.yml
     working_dir: /etc/nordderper
+    healthcheck:
+      test: "curl http://localhost:8765"
```

Comment:
Hmm, maybe some comment why 8765? (Are these just descending digits? ^^' If so, maybe 1234 is more obvious)

Reply:
8765 is configured derp port https://github.com/NordSecurity/libtelio/blob/main/nat-lab/data/nordderper/config1.yml#L3

Reply:
I will add comment which points to `nat-lab/data/nordderper/config*.yml`

Reply:
Added comment